maproom.py

```diff
@@ -61,11 +61,31 @@ def data_tiles(tz, tx, ty, variable):
         decode_times=False
     ).rename({"X": "lon", "Y": "lat"}).isel(T=-1) # Lesson 5
     data.attrs["colormap"] = pingrid.RAINBOW_COLORMAP # Lesson 5 
-    clipping = None
-    resp = pingrid.tile(data, tx, ty, tz, clipping)
+    data.attrs["scale_min"] = data.min().values
+    data.attrs["scale_max"] = data.max().values
+    resp = pingrid.tile(data, tx, ty, tz)
     return resp # Lesson 5 ends
-    
-    
+
+
+# Lesson 8 starts
+@APP.callback(
+    Output("colorbar", "colorscale"),
+    Output("colorbar", "min"),
+    Output("colorbar", "max"),
+    Input("variable", "value"),
```

Comment:
Here you do not use variable input to select a variable from within a ds. If you are using just a single data variable then maybe there is no use for having this as a callback. But if you want to update the colorbar according to different variable data, refer to the [most recent commit I pushed](https://github.com/iridl/lms-monthly/pull/5/commits/16c40887b2cfb9b38395013d3283daac47ca3b52)

Reply:
The 2nd variable is coming a bit later in the Tuto. Could probably have done this a different way...